src/gui/generalsettings.cpp

```diff
@@ -218,7 +231,7 @@ GeneralSettings::GeneralSettings(QWidget *parent)
 
     // OEM themes are not obliged to ship mono icons, so there
     // is no point in offering an option
-    _ui->monoIconsCheckBox->setVisible(Theme::instance()->monoIconsAvailable());
+    _ui->monoIconsCheckBox->setVisible(cfgFile.isVisible());
```

Comment:
This breaks mono icons -- why has this been changed?

Reply:
Removed this change